core/src/main/java/org/elasticsearch/action/bulk/TransportBulkAction.java

```diff
@@ -144,6 +145,7 @@ protected void doExecute(Task task, BulkRequest bulkRequest, ActionListener<Bulk
                     }
                     createIndexRequest.cause("auto(bulk api)");
                     createIndexRequest.masterNodeTimeout(bulkRequest.timeout());
+                    createIndexRequest.waitForActiveShards(ActiveShardCount.NONE);
```

Comment:
Because the bulk request would have to wait for the same amount of shards anyway during write (it will retry if the shards aren't available)